docs/ref/utils.txt

```diff
@@ -383,49 +465,79 @@ https://web.archive.org/web/20110718035220/http://diveintomark.org/archives/2004
 
             The ``stylesheets`` argument was added.
 
+    ``add_item()``
+    --------------
+
     .. method:: add_item(title, link, description, author_email=None, author_name=None, author_link=None, pubdate=None, comments=None, unique_id=None, categories=(), item_copyright=None, ttl=None, updateddate=None, enclosures=None, **kwargs)
 
         Adds an item to the feed. All args are expected to be strings except
         ``pubdate`` and ``updateddate``, which are ``datetime.datetime``
         objects, and ``enclosures``, which is a list of ``Enclosure`` instances.
 
+    ``num_items()``
+    ---------------
+
     .. method:: num_items()
 
+    ``root_attributes()``
+    ---------------------
+
     .. method:: root_attributes()
 
         Return extra attributes to place on the root (i.e. feed/channel)
         element. Called from ``write()``.
 
+    ``add_root_elements()``
+    -----------------------
+
     .. method:: add_root_elements(handler)
 
         Add elements in the root (i.e. feed/channel) element.
         Called from ``write()``.
 
+    ``add_stylesheets()``
+    ---------------------
+
     .. method:: add_stylesheets(self, handler)
 
         .. versionadded:: 5.2
 
         Add stylesheet information to the document.
         Called from ``write()``.
 
+    ``item_attributes()``
+    ---------------------
+
     .. method:: item_attributes(item)
 
         Return extra attributes to place on each item (i.e. item/entry)
         element.
 
+    ``add_item_elements()``
+    -----------------------
+
     .. method:: add_item_elements(handler, item)
 
         Add elements on each item (i.e. item/entry) element.
 
+    ``write()``
+    -----------
+
     .. method:: write(outfile, encoding)
 
         Outputs the feed in the given encoding to ``outfile``, which is a
         file-like object. Subclasses should override this.
 
+    ``writeString()``
+    -----------------
+
     .. method:: writeString(encoding)
 
         Returns the feed in the given encoding as a string.
 
+    ``latest_post_date()``
+    ----------------------
```

Comment:
All of these headers - those within the `SyndicationFeed` class section, have the wrong type of character used to underline the header. They should be tildes, like `~~~~~`. In the rest of the document, `-----` is correct, but as we already have a header for the class using the `-----` style in this part, the headers 'under' this header and part of this section should use `~~~~~`.

For an explanation/info you can check [the contributing documentation on writing docs](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files) which shows the different header styles to be used for each level.